stage-3--rfc-template.md

```diff
@@ -10,9 +10,12 @@
   of the accepted proposal.
 -->
 
+**If you have feedback and the feature is released as experimental, please leave it on the Stage 3 PR. Otherwise, comment on the Stage 2 issue (links below).**
```

Comment:
There's no link to the Stage 2 RFC below actually...should we add it to the template as well? More links can't hurt I guess

Reply:
It's already here

https://github.com/withastro/roadmap/blob/0de5839720b0bb7bfe2d6c8736df69eff1262be9/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L16


I think it's enough

Reply:
I don't know if it's enough. I opened this PR because people were confused where to leave feedback. If we tell them to go to the Stage 2 RFC but they have to go through the Stage 3 PR, it's not gonna help much imo

Reply:
Got it. No problem! I don't mind adding more links, if they help